arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1146 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Operational InsightsManagement Client for LinkTargets, StorageInsightConfigs and SavedSearches.",
+    "description": ".Net client wrapper for the REST API for Azure Operational Insights",
+    "version": "2015-03-20"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
```

Comment:
This helps us in exposing you API over [here](https://azure.github.io/projects/apis/) once the PR is merged. Once you click on Authorize you can login and try those APIs defined in swagger.